lib/timeline.js

```diff
@@ -31,7 +31,8 @@ Timeline.Pin = Pin;
  */
 Timeline.prototype.sendUserPin = function (userToken, pin, callback) {
   if (typeof userToken !== 'string') {
-    return callback(new Error('Expected userToken to be a string.'));
+    process.nextTick(callback(new Error('Expected userToken to be a string.')));
```

Comment:
this will still execute the callback immediately. we'll need smth like

```js
return process.nextTick(function(){
  callback(new Error(..))
});
```